Python/initconfig.c

```diff
@@ -296,6 +296,7 @@ static const char usage_envvars[] =
 "PYTHON_COLORS           : If this variable is set to 1, the interpreter will"
 "   colorize various kinds of output. Setting it to 0 deactivates this behavior.\n"
 "These variables have equivalent command-line parameters (see --help for details):\n"
+"PYTHON_HISTORY: the location of a .python_history file.\n"
```

Comment:
What command line parameter sets it?

Reply:
It doesn't have one, I compared with `PYTHON_COLORS` which doesn't either (https://github.com/python/cpython/pull/112732). Should we add one?

Reply:
If it doesn't have one, then it is in the wrong section.

Reply:
Good point, please see PR https://github.com/python/cpython/pull/113798 to move it to the correct section.